docs/advanced_topics/Tags.md

```diff
@@ -0,0 +1,145 @@
+(tagging)=
```

Comment:
Please use lower case filenames, this should be called `tags.md`.

Reply:
To change this, I created two commits because Git is case-sensitive by default, but some file systems are case-insensitive. It wasn’t differentiating between` Tags.md` and `tags.md`. So, I first renamed it from `Tags.md `to `temp.md` and then to `tags.md`.

Reply:
Thanks.